app/views/components/_result-card.html.erb

```diff
@@ -28,6 +28,7 @@
   data: {
     "track-category": "SmartAnswerClicked",
     "track-action": url_track_action,
-    "track-label": url
+    "track-label": url,
+    "ecommerce-path": url
```

Comment:
Could this be more specifically related to the GA4 code? Something like `ga4-ecommerce-path`?